CsCore/xUnitTests/xUnitTests.csproj

```diff
@@ -25,6 +25,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+    <PackageReference Include="StbImageWriteSharp" Version="1.16.7" />
+    <PackageReference Include="System.Drawing.Common" Version="8.0.1" />
```

Comment:
`System.Drawing.Common` is a problem if you rely on it, since its not available on all platforms like Unity, where are you using this in your algorithoms, maybe we find a way to replace it

Reply:
I believe we added this at some point for loading and writing images in the unit tests, as we didn't understand how it works with the csutil functionality. Now all tests use the existing way, so this dependency is just a leftover. We apologize for the confusion